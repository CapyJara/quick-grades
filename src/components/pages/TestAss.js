import React from 'react';

const TestAss = () => {
  return (
    <>
      <h1>Sorry</h1>
      <h3>You cant actually look at students grades if you&quot;re using the &ldquo;TEST&ldquo; api key</h3>
      <button onClick={() => window.close()}>Close this winda</button>
    </>
  );
};

export default TestAss;
