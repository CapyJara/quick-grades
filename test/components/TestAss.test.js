import React from 'react';
import { shallow } from 'enzyme';
import TestAss from '../../src/components/pages/TestAss';

describe('TestAss component', () => {
  it('renders TestAss', () => {
    const wrapper = shallow(<TestAss />);
    expect(wrapper).toMatchSnapshot();
  });
});
