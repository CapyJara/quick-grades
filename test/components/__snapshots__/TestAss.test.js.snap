// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestAss component renders TestAss 1`] = `
<Fragment>
  <h1>
    Sorry
  </h1>
  <h3>
    You cant actually look at students grades if you"re using the “TEST“ api key
  </h3>
</Fragment>
`;
